tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p>Fix the spacing between <code>working with Qiskit.  The fundamental package</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Just fixed it. Thanks Soolu!</p>